File: starter_kit/bots/csharp/example_project/MyBot.cs
using System;
using Pirates;


namespace Bots
{
    public class MyBot : Pirates.IPirateBot
    {
        public void DoTurn(IPirateGame game)
        {
            if (MyBotUtils.IsTurnMultipleOf100(game)) {
                game.Debug("Turn number divides in 100.");
            }
        }

    }
}

File: starter_kit/bots/java/example_project/bots/MyBot.java
package bots;

import pirates.game.PirateBot;
import pirates.game.PirateGame;


public class MyBot implements PirateBot{

    @Override
    public void doTurn(PirateGame game) {
        // Demonstrating usage of another file
        if (MyBotUtils.isTurnMultipleOf100(game)) {
            game.debug("Turn divides by 0");
        }
    }

}
